tamr_unify_client/mastering/published_cluster/resource.py

```diff
@@ -0,0 +1,33 @@
+from tamr_unify_client.mastering.published_cluster.version import (
+    PublishedClusterVersion,
+)
+
+
+class PublishedCluster:
+    """
```

Comment:
super nit-picky but there shouldn't be a new line between """ and A representation

ie 

""" A representation of a...